### PR TITLE
Services class exposes tags for given service name

### PR DESCRIPTION
**Pull request by arkadiuszbielewicz**

### CODE REVIEW

File: src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java [hunk 1]
```diff
@@ -12,11 +13,11 @@ public Services(Map<String, List<String>> serviceNamesToTags) {
         this.serviceNamesToTags = serviceNamesToTags;
     }
 
-    boolean containsService(String serviceName) {
+    public boolean containsService(String serviceName) {
```
Comment (malfoj89) [inline src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java:16]: po co ta metoda ma byc publiczna?

File: src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java [hunk 2]
```diff
@@ -27,6 +28,10 @@ public Set<String> serviceNames() {
         return serviceNamesToTags.keySet();
     }
 
+    public List<String> tagsForServiceOrNull(String serviceName) {
```
Comment (malfoj89) [inline src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java:31]: po co dodatkowa metoda?

**Review by Ferdudas97 — APPROVED** [review]

File: src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java (hunk 2 above)
Comment (arkadiuszbielewicz) [reply src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java:31]: Throwing exceptions is less performant, whereas `null` case is recoverable in my case :)

File: src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java (hunk 1 above)
Comment (arkadiuszbielewicz) [reply src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java:16]: > po co ta metoda ma byc publiczna? (why is this method supposed to be public?)
Services is simple wrapper around Map, I don't see any point in hiding such methods which might be useful in some context.

**Pull request merged**